### PR TITLE
Fifth chunk of Cobra Examples

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

The fifth and final chunk of examples for the Cobra
examples in the CLI help output.

Also includes a few man page touchups.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

File: cmd/podman/generate_kube.go [hunk 1]
```diff
@@ -25,7 +25,9 @@ var (
 			containerKubeCommand.GlobalFlags = MainGlobalOpts
 			return generateKubeYAMLCmd(&containerKubeCommand)
 		},
-		Example: "CONTAINER|POD-NAME",
+		Example: `podman generate kube ctrID
+  podman generate kube podID
+  podman generate kube --service podID`,
```
Comment (mheon) [inline cmd/podman/generate_kube.go]: @baude Didn't we remove `--service`?

File: cmd/podman/logs.go [hunk 2]
```diff
@@ -26,7 +26,9 @@ var (
 			logsCommand.GlobalFlags = MainGlobalOpts
 			return logsCmd(&logsCommand)
 		},
-		Example: "CONTAINER",
+		Example: `podman logs ctrID
+  podman logs --tail 2 mywebserver
+  podman logs --since 10m ctrID`,
```
Comment (mheon) [inline cmd/podman/logs.go]: Maybe --follow?

File: cmd/podman/generate_kube.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/generate_kube.go]: It's still in the man page and looks to be in the code too:
```
        if c.Bool("service") {
                output = append(output, []byte("---\n")...)
                output = append(output, marshalledService...)
        }
```

File: cmd/podman/logs.go (hunk 2 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/logs.go]: added

**Pull request merged**